### PR TITLE
Initial support for more openwrt, more oe, or neutral flavours of meta-openwrt

### PR DESCRIPTION
**Pull request by cshoredaniel**
This is some initial PACKAGECONFIG work to allow to choose a more openwrt-like image, a neutral (the default) which has some openwrt-specific things, but is not as close to upstream openwrt, and more oe-based (more for picking out specific packages from meta-openwrt without requiring too much from openwrt just to do that).

### CODE REVIEW

**Comment by cshoredaniel** [comment]
ping?  comments?

File: recipes-tweaks/busybox/busybox_1.27%.bbappend [hunk 1]
```diff
@@ -5,12 +5,16 @@
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+PACKAGECONFIG ??= ""
+PACKAGECONFIG[preferoe] = ""
```
[the remark this answers is not in the document]
Comment (cshoredaniel) [reply recipes-tweaks/busybox/busybox_1.27%.bbappend]: You mean make the default maximally openwrtish?

[the remark this answers is not in the document]
Comment (cshoredaniel) [reply recipes-tweaks/busybox/busybox_1.27%.bbappend]: Or call 'includeopenwrt' 'openwrt'?

**Comment by kraj** [comment]
something related to openwrt name.. but you can submit as a follow up

**Pull request merged**